Add Defender for Endpoint File Indicator to file statistics command (#37022)
demisto/content: Packs/CommonTypes/ReleaseNotes/3_6_0.md

> 

Reviewer: Please fix these according to the description changes above as well.

#### Indicator Fields

##### New: Organization Prevalence

Added new number field to indicate how frequently a specific indicator type is detected within a particular organization.

##### New: Global Prevalence

Added new number field to indicate how frequently a specific indicator type is detected across all organizations.

##### New: Organization First Seen

Added new date field to indicate the first time a specific organization encountered an indicator.

##### New: Organization Last Seen

> Added new date field to indicate the most recent a specific organization encountered an indicator.

Reviewer 2: ```suggestion
Added new date field to indicate the most recent time a specific organization encountered an indicator.
```

Author: Well spotted! Fixed

##### New: Global First Seen

Added new date field to indicate the first time an indicator was detected globally across all organizations.

##### New: Global Last Seen

Added new date field to indicate the most recent time an indicator was detected globally across all organizations.

#### Indicator Types

##### File

> Added default mapping to new indicator fields.

Reviewer: Add the fields that were mapped as well.

Author: Added in commit db38beed6372507e3cf58090eb33520e7c27dfea

#### Indicator Fields

##### New: Organization Prevalence

> Added new number field which represents how frequently a specific indicator type is detected within a particular organization.

Reviewer 2: ```suggestion
Added new number field which represents the number of times a specific indicator type is detected within a particular organization.
```

Author: Fixed, thanks

> Added new number field which represents how frequently a specific indicator type is detected within a particular organization.

Author: Fixed after discussing with @ShirleyDenkberg

##### New: Global Prevalence

> Added new number field which represents how frequently a specific indicator type is detected across all organizations.

Reviewer 2: ```suggestion
Added new number field which represents the number of times a specific indicator type is detected across all organizations.
```

Author: Fixed
